src/Resources/config/routing/api.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<routes xmlns="http://friendsofsymfony.github.com/schema/rest" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://friendsofsymfony.github.com/schema/rest https://raw.github.com/FriendsOfSymfony/FOSRestBundle/master/Resources/config/schema/routing/rest_routing-1.0.xsd">
-    <import type="rest" resource="Sonata\PageBundle\Controller\Api\BlockController" name-prefix="sonata_api_block_"/>
-    <import type="rest" resource="Sonata\PageBundle\Controller\Api\PageController" name-prefix="sonata_api_page_"/>
-    <import type="rest" resource="Sonata\PageBundle\Controller\Api\SiteController" name-prefix="sonata_api_site_"/>
-    <import type="rest" resource="Sonata\PageBundle\Controller\Api\SnapshotController" name-prefix="sonata_api_snapshot_"/>
+<routes xmlns="http://symfony.com/schema/routing" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/routing https://symfony.com/schema/routing/routing-1.0.xsd">
+    <import type="xml" resource="/api/block.xml" name-prefix="sonata_api_block_"/>
```

Comment:
In [other PRs](https://github.com/sonata-project/SonataNewsBundle/pull/612/files#diff-510d26ba1c2330a6a6450f6fa81f0466R3) you're using paths starting with `./` instead of `/`. Please, try to be consistent about that. Regardless it could be working, a path starting with `/` could be interpreted as absolute, so I think we should try to avoid that. If paths like `api/block.xml` or `./api/block.xml` are working, I prefer to use these variants.

Reply:
2 lines under i made bug with `.xmlr`.
Fixed